boards/nxp/imx95_evk/CMakeLists.txt

```diff
@@ -12,5 +12,104 @@ if (CONFIG_SOF AND CONFIG_BOARD_IMX95_EVK_MIMX9596_M7_DDR)
+      "
+      family: mimx9596
+      revision: a1
+      target_memory: standard
+      output:                   ${CMAKE_BINARY_DIR}/zephyr/primary_ahab.bin
+      containers:
+        - binary_container:
+            path:               ${CMAKE_BINARY_DIR}/zephyr/${FIRMWARE_RELEASE}/mx95a0-ahab-container.img
+        - container:
+            srk_set: none
+            images:
+              - lpddr_imem:     ${CMAKE_BINARY_DIR}/zephyr/${FIRMWARE_RELEASE}/lpddr5_imem_v202311.bin
+                lpddr_imem_qb:  ${CMAKE_BINARY_DIR}/zephyr/${FIRMWARE_RELEASE}/lpddr5_imem_qb_v202311.bin
+                lpddr_dmem:     ${CMAKE_BINARY_DIR}/zephyr/${FIRMWARE_RELEASE}/lpddr5_dmem_v202311.bin
+                lpddr_dmem_qb:  ${CMAKE_BINARY_DIR}/zephyr/${FIRMWARE_RELEASE}/lpddr5_dmem_qb_v202311.bin
+                oei_ddr:        ${CMAKE_BINARY_DIR}/zephyr/${FIRMWARE_RELEASE}/oei-m33-ddr.bin
+              - oei_tcm:        ${CMAKE_BINARY_DIR}/zephyr/${FIRMWARE_RELEASE}/oei-m33-tcm.bin
+              - system_manager: ${CMAKE_BINARY_DIR}/zephyr/${FIRMWARE_RELEASE}/m33_image-mx95evk.bin
+              - spl:            ${CMAKE_BINARY_DIR}/zephyr/${FIRMWARE_RELEASE}/u-boot-spl.bin-imx95-19x19-lpddr5-evk-sd
+              - v2x_dummy: true
+    ")
+    file(WRITE ${CMAKE_BINARY_DIR}/zephyr/imx95_evk_mimx9596_a55_ahab_secondary.yaml
+      "
+      family: mimx9596
+      revision: a1
+      target_memory: standard
+      output:                   ${CMAKE_BINARY_DIR}/zephyr/secondary_ahab.bin
+      containers:
+        - container:
+            srk_set: none
+            images:
+              - atf:            ${ZEPHYR_HAL_NXP_MODULE_DIR}/zephyr/blobs/imx-firmware/imx95-19x19-lpddr5-evk/bl31-imx95-zephyr.bin
+              - image_path:     ${CMAKE_BINARY_DIR}/zephyr/${CONFIG_KERNEL_BIN_NAME}.bin
+                load_address:   '${CONFIG_SRAM_BASE_ADDRESS}'
+                entry_point:    '${CONFIG_SRAM_BASE_ADDRESS}'
+                image_type:     executable
+                core_id:        cortex-a55
+                is_encrypted:   false
+    ")
+    file(WRITE ${CMAKE_BINARY_DIR}/zephyr/imx95_evk_mimx9596_a55_ahab_flash_template.yaml
+      "
+      family: mimx9596
+      revision: latest
+      memory_type: serial_downloader
+      primary_image_container_set: ${CMAKE_BINARY_DIR}/zephyr/imx95_evk_mimx9596_a55_ahab_primary.yaml
+      secondary_image_container_set: ${CMAKE_BINARY_DIR}/zephyr/imx95_evk_mimx9596_a55_ahab_secondary.yaml
+   ")
+    set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
+      COMMAND chmod +x ${ZEPHYR_HAL_NXP_MODULE_DIR}/zephyr/blobs/imx-boot-firmware/${FIRMWARE_RELEASE}.bin
+      COMMAND ${ZEPHYR_HAL_NXP_MODULE_DIR}/zephyr/blobs/imx-boot-firmware/${FIRMWARE_RELEASE}.bin --auto-accept --force
+      COMMAND nxpimage -v bootable-image merge -c ${CMAKE_BINARY_DIR}/zephyr/imx95_evk_mimx9596_a55_ahab_flash_template.yaml -o flash.bin
```

Comment:
this command line is change to `nxpimage -v bootable-image export` see M7 above

Reply:
Confirmed from SPSDK v3.0.0 it will change from merge command to export command, so added version checking to adapt to different SPSDK version, thanks.